locale/en/blog/community/quality-with-speed.md

```diff
@@ -0,0 +1,316 @@
+  * Platform/os coverage
+  * Development Workflows
+* Performance Benchmarks
+* Tools
+
+# Release Types
+
+The Node.js project maintains 3 key types of releases
+* Nightlies
+* Current
+* LTS
+
+Having different release types allows innovation/change
+to flow rapidly into Nightly releases where they can bake.
+When ready, these changes then transition
+to Current and LTS releases in a more controlled manner,
+such that the level of quality/stability increases at
+each level.
+
+## Nightlies
+
+These are built from master and contain the very latest changes
+that have been landed. If you want to try out the bleeding edge
+these are the binaries to use.  There is no additional testing
+on these releases, however, the standard Node.js unit tests are
+run for each change landed so these will most often be usable.
+
+## Current
+
+Changes which have landed in master are backported to Current
+on a regular basis. In general all changes that land in master
+will be backported to Current, however there may be a lag if
+there are specific concerns or for larger changes where the community
+believes more soak time is required.  One key exception is
+that sermver breaking changes will not be backported until the
+next major version (ex 5 -> 6).  This includes v8 and other
+components such that the expectation is that an application/module
+written to run on a major level will continue to do so.
+
+These releases are documented in the changelog so
+there is more visibility with respect to the changes in the release.
+Current releases are created roughly every 1-2 weeks.
+
+In addition to the the regular Node.js unit tests, CITGM(see
+later sections) is run on Current releases.
+
+If you want to try out the latest with a reasonable expectation
+that your application will continue to run, these are the releases
+to use.
+
+## LTS
+
+Once changes have been proven in the Current stream, they are candidates
+for the LTS stream.  Changes are limited to:
+
+* Bug fixes
+* Security updates
+* Non-semver-major npm updates
+* Relevant documentation updates
+* Certain performance improvements where the risk of
+  breaking existing applications is minimal
+* Changes that introduce large amount of code churn where
+  the risk of breaking existing applications is low and
+  where the change in question may significantly ease the
+  ability to backport future changes due to the reduction in diff noise.
+
+Further, in the later half of an LTS release (Maintenance), only
+**critical** bugs and **critical** security fixes will be included.
+
+Like Current releases, CITGM(see
+later sections) is run on Current releases.  In addition we also
+track performance through nightly benchmarks reported on
+[benchmarking.nodejs.org](benchmarking.nodejs.org) (See later sections).
+
+You can read more about the LTS releases [here](https://github.com/nodejs/lts).
+
+If you want the best level of stability/quality for your production
+applications these are the releases to use.
```

Comment:
but you should pay close attention to their end-of-life dates, some don't last very long!

it may be worth noting what support (none) node offers once LTS is out of maintenance